CHANGELOG.md

```diff
@@ -5,30 +5,55 @@ Changelog for NeoFS Node
 
 ### Added
 
+### Changed
+
+### Fixed
+
+- Losing request context in eACL response checks (#1595)
+
+### Removed
+
+### Updated
+
+### Updating from v0.30.0
+
+## [0.30.0] - 2022-07-22 - Saengildo (생일도, 生日島)
+
+### Added
+
 - Profiler and metrics services now should be enabled with a separate flag
 - Experimental support for the tree-service, disabled by default (#1607)
+- Homomorphic hashes calculation can be disabled across the whole network (#1365)
+- Improve `neofs-adm` auto-completion (#1594)
 
 ### Changed
 
 - Require SG members to be unique (#1490)
 - `neofs-cli` now doesn't remove container with LOCK objects without `--force` flag (#1500)
+- LOCK objects are now required to have an expiration epoch (#1461)
 - `morph` sections in IR and storage node configuration now accept an address and a priority of an endpoint (#1609)
 - Morph client now retries connecting to the failed endpoint too (#1609)
+- Redirecting `GET` and `GETRANGE` requests now does not store full object copy in memory (#1605)
+- `neofs-adm` now registers candidates during initialization in a single transaction (#1608)
 
 ### Fixed
 
-- Losing request context in eACL response checks (#1595)
-
-### Removed
+- Shard now can start in degraded mode if the metabase is unavailable (#1559)
+- Shard can now be disabled completely on init failure (#1559)
+- Storage group members are now required to be unique (#1490)
+- Print shard ID in component logs (#1611)
 
 ### Updated
+- `neofs-contract` to `v0.15.3`
+- `neo-go` to the pre-release version
+- `github.com/spf13/cobra` to v1.5.0
 
 ### Updating from v0.29.0
 Change morph endpoints from simple string to a pair of `address` and `priority`. The second can be omitted.
 For inner ring node this resides in `morph.endpoint.client` section,
 for storage node -- in `morph.rpc_endpoint` section. See `config/example` for an example. 
```

Comment:
also about metrics and pprof? (optional renaming and explicit toggle via `enable`)
moving shards defaults to the `shard` section?

Reply:
Fixed